src/test/resources/relationship-tests.adoc

```diff
@@ -0,0 +1,323 @@
+:toc:
+
+= Movie Test TCK
```

Comment:
Should this be `Relationship Test TCK` rather than `Movie Test TCK`?

Reply:
Thanks for spotting this, fixed in master.